integration-tests/src/tests/client/process_blocks.rs

```diff
@@ -3459,6 +3459,96 @@ fn test_catchup_no_sharding_change() {
     }
 }
 
+/// Tests if the cost of deployment is higher after the protocol update 53
+#[test]
+fn test_deploy_cost_increased() {
+    let old_protocol_version = 52;
+    let new_protocol_version = 53;
```

Comment:
Alternatively I would introduce a protocol feature like `IncreaseDeploymentCost` and use its protocol version (- 1).

I think of a scenario when we can postpone this feature to the v54 due to emergency reasons. Then it will be natural to modify protocol version of feature in the `version.rs`, but it can be easy to forget to fix two numbers in some test.

Reply:
You are right, I have now added this as an explicit protocol feature.